src/caret_analyze/runtime/path.py

```diff
@@ -210,37 +208,48 @@ def is_source_timestamp_column(column: str) -> bool:
                 )
 
         if include_last_callback and isinstance(targets[-1], NodePath):
-            right_records = targets[-1].to_path_end_records()
-
-            rename_rule = column_merger.append_columns_and_return_rename_rule(right_records)
-            right_records.rename_columns(rename_rule)
-            if left_records.columns[-1] != right_records.columns[0]:
-                raise InvalidRecordsError('left columns[-1] != right columns[0]')
-            if len(right_records.data) != 0:
-                left_records = merge(
-                    left_records=left_records,
-                    right_records=right_records,
-                    join_left_key=left_records.columns[-1],
-                    join_right_key=right_records.columns[0],
-                    columns=Columns.from_str(
-                        left_records.columns + right_records.columns
-                    ).column_names,
-                    how='left'
-                )
+            if not is_match_column(left_records.columns[-1], 'source_timestamp'):
+                right_records = targets[-1].to_path_end_records()
+
+                rename_rule = column_merger.append_columns_and_return_rename_rule(right_records)
+                right_records.rename_columns(rename_rule)
+                if left_records.columns[-1] != right_records.columns[0]:
+                    raise InvalidRecordsError('left columns[-1] != right columns[0]')
+                if len(right_records.data) != 0:
+                    left_records = merge(
+                        left_records=left_records,
+                        right_records=right_records,
+                        join_left_key=left_records.columns[-1],
+                        join_right_key=right_records.columns[0],
+                        columns=Columns.from_str(
+                            left_records.columns + right_records.columns
+                        ).column_names,
+                        how='left'
+                    )
+                else:
+                    msg = 'Empty records are not merged.'
+                    logger.warn(msg)
             else:
-                msg = 'include_last_callback argument is ignored '
-                msg += 'because last node receive messages '
-                msg += 'by `take` method instead of subscription callback.'
+                msg = 'If the last record is a take implementation, '
+                msg += 'the path cannot be extended, so the merge process is skipped.'
```

Comment:
This sentence might lead users to misunderstand that the entire path merge is being skipped. Therefore, how about explicitly stating that the merge is skipped for a specific target (last_callback records), like this: 'the path cannot be extended, so the merge process for last_callback records is skipped.'?